client/src/api/ingredients.js

```diff
@@ -0,0 +1,7 @@
+import { fetchAll } from './helpers';
+
+const urls = {
+  root: '/ingredients',
+};
+
+export default { fetchAll: () => fetchAll(urls.root) };
```

Comment:
this kind of one liners tends to get extremely hard to read when there's a lot of exports. Can you remove the function from the export and just export it like it's done on recipes